packages/core-js/utils/misc/unique.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Helper array unique function to be used with Array.filter method.
+ *   e.g.: [1, 1, 2, 3].filter(unique) // => [1, 2, 3]
+ */
+module.exports = function unique(value, index, self) {
+  return self.indexOf(value) === index;
+};
```

Comment:
We've been using this pattern for getting the unique values from an array which IMHO is kinda difficult to read, so, I just moved it to an util which makes it more readable when using it.